validation.go

```diff
@@ -51,6 +54,9 @@ const (
 
 	// FormatRFC1123 defines RFC1123 date time values.
 	FormatRFC1123 = "rfc1123"
+
+	// LayoutRFC3339Date is a layout of RFC3339 date for use in time.Parse.
+	LayoutRFC3339Date = "2006-01-02"
```

Comment:
Do we really need a documented constant here? wondering if just an inline string in two places isn't simpler - this is not likely to change ever...

Reply:
I agree that an inline string is better for this use case.